features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/App.js

```diff
@@ -47,11 +47,30 @@ const Logout = () => import(/* webpackChunkName: "logout" */ './app/components/L
 class Protected extends Component {
     constructor(props) {
         super(props);
-        this.state = {showLeftMenu: false};
+        this.state = {
+            showLeftMenu: false,
+            authConfigs: null
+        };
         message.config({top: '48px'}); // .custom-header height + some offset
         /* TODO: need to fix the header to avoid conflicting with messages ~tmkb*/
     }
 
+    handleResponse = (response) => {
```

Comment:
same as mentioned above, this is an ES7 syntax which is currently in proposal stage 3

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/4890/commits/57e9929cdc68c5b4447681c94a793b86b0390380